src/SFA.DAS.Provider.PR.Web/Controllers/EmployerDetailsController.cs

```diff
@@ -54,4 +61,13 @@ public async Task<IActionResult> Index([FromRoute] int ukprn, [FromRoute] Guid r
 
         return View(model);
     }
+
+    private static bool IsActivePermissionsRequest(Response<GetRequestsByRequestIdResponse> response)
+    {
+        var responseContent = response.GetContent();
+
+        return string.Equals(responseContent.RequestType, RequestType.Permission.ToString(), StringComparison.CurrentCultureIgnoreCase) &&
```

Comment:
Since this is controlled response from inner api, there should not be a need to check case.